### PR TITLE
feat: use filepond to upload files

### PR DESCRIPTION
**Pull request by BlockLune**
**This is an experimental exploration that needs further discussion and validation!**

The current file uploader is ugly and I found FilePond -- a well designed and featured file uploader.

However, I haven't completely figured out how to use this component in this Nuxt project, and it's also a question of how to adjust the style of this component to harmonize it with the overall project style.

Maybe the best and easiest way is waiting https://github.com/nuxt/ui/issues/1422?

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+      credits="false"
+      accept="image/*"
+      :label-idle="
+        $t('upload.filepond.labelIdle.part1') +
+        '<span class=\'filepond--label-action\'>' +
+        $t('upload.filepond.labelIdle.part2') +
+        '</span>'
+      "
+    />
     <UTabs
```
Comment (yy4382) [inline pages/index.vue:15]:
```suggestion
    <ClientOnly>
      <FilePond
        allowMulti="true"
        credits="false"
        accept="image/*"
        :label-idle="
          $t('upload.filepond.labelIdle.part1') +
          '<span class=\'filepond--label-action\'>' +
          $t('upload.filepond.labelIdle.part2') +
          '</span>'
        "
      />
    </ClientOnly>
```
maybe not ssr compatible

File: pages/index.vue [hunk 2]
```diff
@@ -184,4 +188,7 @@ const uploadHandler = async (e: any) => {
   }
   uploading.value = false;
 };
+
+import createVueFilePond from "vue-filepond";
+const FilePond = createVueFilePond();
```
Comment (yy4382) [inline pages/index.vue:193]: when deployed to vercel, it complains that "createVueFilePond is not a function", and throws an 500 error.

File: pages/index.vue (hunk 1 above)
Comment (yy4382) [reply pages/index.vue:15]: oh, some mistakes on choosing the area

File: pages/index.vue [hunk 3]
```diff
@@ -59,7 +63,7 @@
 
 <script setup lang="ts">
 import { DateTime, Interval } from "luxon";
-import { useStorage } from "@vueuse/core";
+import { get, useStorage } from "@vueuse/core";
```
Comment (yy4382) [inline pages/index.vue:66]: a mistake of auto-import? I didn't see where `get` is used.

Comment (BlockLune) [reply pages/index.vue:66]: Maybe

**Comment by yy4382** [comment]
Won't use filepond. Let's build from scratch!

**Pull request closed without merging**